Patches/DMWands/Patch_Weapons.xml

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>D4egons Magic Wands</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[
+				defName="DM_BasicMagicWand" or
+				defName="DM_LesserMagicWand" or
+				defName="DM_MagicWand" or
+				defName="DM_GreaterMagicWand" or
+				defName="DM_FireWand" or
+				defName="DM_PoisonWand"
+				]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>wand</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>5</power>
+							<cooldownTime>1.2</cooldownTime>
+							<armorPenetrationBlunt>0.4</armorPenetrationBlunt>
+							<linkedBodyPartsGroup>Shaft</linkedBodyPartsGroup>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>/Defs/ThingDef[defName="DM_MagicBolt"]</xpath>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs</xpath>
+				<value>
+					<ThingDef Class="CombatExtended.AmmoDef" ParentName="AmmoBase">
```

Comment:
For all these wand projectiles, remove `Class="CombatExtended.AmmoDef"` and set the Parent to `BaseBullet`. AmmoBase is for the ammo item, not the projectile.